code_samples/action_basics/src/main/java/org/intellij/sdk/action/PopDialogAction.java

```diff
@@ -0,0 +1,78 @@
+// Copyright 2000-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
+
+package org.intellij.sdk.action;
+
+import com.intellij.openapi.actionSystem.AnAction;
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import com.intellij.openapi.actionSystem.CommonDataKeys;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.ui.Messages;
+import com.intellij.pom.Navigatable;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import javax.swing.*;
+
+/**
+ * Action class to demonstrate how to interact with the IntelliJ Platform.
+ * The only action this class performs is to provide the user with a popup dialog as feedback.
+ * Typically this class is instantiated by the IntelliJ Platform framework based on declarations
+ * in the plugin.xml file. But when added at runtime this class is instantiated by an action group.
+ */
+public class PopDialogAction extends AnAction {
+  
+  /**
+   * This default constructor is used by the IntelliJ Platform framework to
+   * instantiate this class based on plugin.xml declarations. Only needed in PopDialogAction
+   * class because a second constructor is overridden.
+   * @see AnAction#AnAction()
+   */
+  public PopDialogAction() {
+    super();
+  }
+  
+  /**
+   * This constructor is used to support dynamically added menu actions.
+   * It sets the text, description to be displayed for the menu item.
+   * Otherwise, the default AnAction constructor is used by the IntelliJ Platform.
+   * @param menuText  The text to be displayed as a menu item.
+   * @param menuDescription  The description of the menu item.
+   * @param menuIcon  The icon to be used with the menu item.
+   */
+  public PopDialogAction(@Nullable String menuText, @Nullable String menuDescription, @Nullable Icon menuIcon) {
```

Comment:
parameter names should be the same as in `AnAction`

Reply:
Done.